### PR TITLE
Add Image Categories

### LINKED ISSUE #292: Move default categories from API to providers scripts
## Problem

The `category` field should be pre-populated for images as it is done for audio rather than assigned during data refresh [as performed currently](https://github.com/WordPress/openverse-api/blob/e0949d2641f20506338dc6a40250c24f5a8fa517/ingestion_server/ingestion_server/categorize.py). So default categories need to be transferred here to be used in each corresponding provider API script.

## Additional context

New sample data for images, containing the `category` column would be needed in the API. Reference to WordPress/openverse-api#357.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Related #292 by @dhruvkb

## Description

This PR copy the default categories for images from the API to be used by the Provider API scripts. It uses these categories to complete the Flickr and StockSnap scripts and then fills the `filesize`, `filetype` and `category` columns.

I'm not sure where the rest of the sources come from (from which provider) so it seems to me that the use of the rest of the categories should be incremental as we test each script and fill these columns also. Flickr and StockSnap are selected in order to have more complete sample data for the API.

## Testing Instructions

1. Run the project and from the Airflow UI run these three dags: `flick_workflow`, `stocksnap_workflow` and `tsv_to_postgres_loader`.
2. Wait for some data to be collected.
3. Check for the fields filled in the database.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: openverse_catalog/dags/common/loader/provider_details.py [hunk 1]
```diff
@@ -105,3 +107,30 @@
...
+MCCORDMUSEUM_DEFAULT_IMAGE_CATEGORY = ImageCategory.DIGITIZED_ARTWORK.value
+MET_DEFAULT_IMAGE_CATEGORY = ImageCategory.DIGITIZED_ARTWORK.value
+MUSEUMSVICTORIA_DEFAULT_IMAGE_CATEGORY = ImageCategory.DIGITIZED_ARTWORK.value
+PHYLOPIC_DEFAULT_IMAGE_CATEGORY = ImageCategory.ILLUSTRATION.value
+RIJKSMUSEUM_DEFAULT_IMAGE_CATEGORY = ImageCategory.DIGITIZED_ARTWORK.value
+SCIENCEMUSEUM_DEFAULT_IMAGE_CATEGORY = ImageCategory.PHOTOGRAPH.value
+STOCKSNAP_DEFAULT_IMAGE_CATEGORY = ImageCategory.PHOTOGRAPH.value
+SVGSILH_DEFAULT_IMAGE_CATEGORY = ImageCategory.ILLUSTRATION.value
+THORVALDSENMUSEUM_DEFAULT_IMAGE_CATEGORY = ImageCategory.DIGITIZED_ARTWORK.value
+WORMS_DEFAULT_IMAGE_CATEGORY = ImageCategory.PHOTOGRAPH.value
```
Comment (AetherUnbound) [inline openverse_catalog/dags/common/loader/provider_details.py:136]: Would this be better as a dictionary where the provider name is used as the key?

**Review by zackkrida — APPROVED** [review]
I _love_ that you updated some of the provider scripts, this is all working for me so I want to approve but I'll let other folks review for things I'm not great at, like python code style and things like that 😅

**Comment by sarayourfriend** [comment]
Reviewing this now but I need to get API keys for flickr and stocksnap so it might take a little while for me to finish reviewing.

I will say that the code looks g2g for me 👍

**Review by sarayourfriend — APPROVED** [review]
Okay, I was able to successfully run the DAGs mentioned in the testing instructions locally and when I query my local DB there are 0 records with an empty category column. 

```
openledger> select * from image where category is NULL
+------------+------------+------------+----------------+----------+--------+--------------------+---------------------+-----+-----------+-------+--------+----------+---
------+-----------------+---------+-------------+-------+-----------+------+-------------+-------------------------+---------------------+----------+----------+
| identifier | created_on | updated_on | ingestion_type | provider | source | foreign_identifier | foreign_landing_url | url | thumbnail | width | height | filesize | li
cense | license_version | creator | creator_url | title | meta_data | tags | watermarked | last_synced_with_source | removed_from_source | filetype | category |
|------------+------------+------------+----------------+----------+--------+--------------------+---------------------+-----+-----------+-------+--------+----------+---
------+-----------------+---------+-------------+-------+-----------+------+-------------+-------------------------+---------------------+----------+----------|
+------------+------------+------------+----------------+----------+--------+--------------------+---------------------+-----+-----------+-------+--------+----------+---
------+-----------------+---------+-------------+-------+-----------+------+-------------+-------------------------+---------------------+----------+----------+
SELECT 0
Time: 0.007s
```

Hopefully that means this is working as expected! Anyway there are no errors as far as I could tell so I think as long as someone else has verified functionality this is g2g from my perspective.

**Review by AetherUnbound — APPROVED** [review]
This looks fantastic! I'll try to test it myself later today, but it seems like others have confirmed so don't let that stop a merge :smile: Thanks also for updating the provider scripts!

**Comment by krysal** [comment]
Thanks, everyone! It's actually possible to have images without category @sarayourfriend, in this case with Flickr, not all images are photos so there may be some `NULL`s in that column.

**Pull request merged**